wrangler-service/src/main/java/co/cask/wrangler/RequestExtractor.java

```diff
@@ -100,6 +101,7 @@ public String getContent(String charset) {
    * @param type class to converted to.
    * @return instance of type T as defined by the class.
    */
+  @Nullable
```

Comment:
I am not able to comment there but [this method](https://github.com/data-integrations/wrangler/pull/284/files#diff-04ee9f099bc61a0c38a03a5107fbf374L67) also returns null values. We should add `Nullable` there too.

Reply:
there are many warnings in this class, i'll fix them.